src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+    id: root => root._id,
+    firstName: root => root.firstName,
+    lastName: root => root.lastName,
```

Comment:
You don't actually need to map it. Same goes with `description`, `birthday` and `hobbies`